edx-platform/pearson-vue-theme/lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -171,10 +173,10 @@
                     <tr>
                         <!-- COPYRIGHT -->
                         <td>
-                            &copy; {% now "Y" %} {{ platform_name }}, {% trans "All rights reserved" as tmsg %}{{ tmsg | force_escape }}.<br/>
-                            <br/>
-                            {% trans "Our mailing address is" as tmsg %}{{ tmsg | force_escape }}:<br/>
-                            {{ contact_mailing_address }}
+                          Copyright &copy; {% now "Y" %} Pearson Education, Inc. or its affiliate(s). All rights reserved.
```

Comment:
Is it necessary to add the Copyright text? Isn't enough with &copy;? @luismorenolopera

Reply:
So it was required